src/snackbar/_snackbar.scss

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "../variables";
+
+.mdl-snackbar {
+  position: absolute;
```

Comment:
That is what absolute here is doing. I think I use viewport/body interchangeably when I shouldn't.

Main thing is this with fixed...

Have a relative/absolute box in the content area somewhere. If you add `fixed` here and put the markup for the snackbar into that, it will be fixed to the bottom of that container. Where absolute, is always positioning it at the bottom of the screen, no matter the parent positioning.

I'll take another look at the spec and test to verify I am not confused about my previous testing results.